apps/frontend/src/components/listingCard.tsx

```diff
@@ -106,9 +116,9 @@ const ListingCard = ({ searchPerformed, listing }: ListingCardProps) => {
               textTransform: "none",
               width: "calc(100% - 20px)",
             }}
-            onClick={handleNotInterested}
+            onClick={canEdit ? handleCanEdit : handleNotInterested}
           >
-            Not interested
+            {canEdit ? "Edit" : "Not interested"}
```

Comment:
I think this is one of the times where more code is better, having an "Edit" vs "Not Interested" button might be more semantic. It also might be better for the user experience if the button for stopping suggestions on someone else's listing vs editing their own listing are different colours. Definitely doesn't need changing, but something to consider.

Reply:
Two different buttons now. 
<img width="1261" alt="Screenshot 2024-06-25 at 7 56 13 AM" src="https://github.com/UVicMartletplace/martletplace/assets/72957712/b4f862cc-b8a2-48e9-aea4-5f403b30915d">